src/Patterns/PatternsHelper.php

```diff
@@ -74,10 +74,11 @@ public static function get_image_url( array $images, int $index, string $default
 
 		return $image;
 	}
+
 	/**
 	 * Returns the post that has the generated data by the AI for the patterns.
 	 *
-	 * @return WP_Post|null
+	 * @return int|\WP_Post|null
```

Comment:
Is there a chance that $post[0] is an int?

Reply:
Nice catch, there isn't! Updated on https://github.com/woocommerce/woocommerce-blocks/pull/11759/commits/0ba01c8e7850d49364c3e84850319fbbd2305b84